### PR TITLE
Fix/issue 780

### PR DESCRIPTION
**Pull request by binwan-dev**
# Description

_Please explain the changes you've made_

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: src/Masa.Stack.Components/wwwroot/css/standard.css [hunk 1]
```diff
@@ -525,3 +525,11 @@
     .masa .full-height {
         height: 100% !important;
     }
+
+    .masa .twelve-m{
```
Comment (MayueCif) [inline src/Masa.Stack.Components/wwwroot/css/standard.css:529]: ?

**Review by MayueCif — APPROVED** [review]
@binwan-dev

File: src/Masa.Stack.Components/wwwroot/css/standard.css (hunk 1 above)
Comment (binwan-dev) [reply src/Masa.Stack.Components/wwwroot/css/standard.css:529]: 设计稿中样式段落名12m，在masa.blazor中搜索到是overline，但是standard.css中overline和官网值不同，故按照设计稿上样式段落名命名

Comment (binwan-dev) [reply src/Masa.Stack.Components/wwwroot/css/standard.css]: 使用 overline、font-weight-medium实现，已经去掉该样式

**Pull request merged**